packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -1397,11 +1414,13 @@ export class SnapController extends BaseController<
    * This is distinct from the state MetaMask uses to manage snaps.
    *
    * @param snapId - The id of the Snap whose state to get.
-   * @returns A promise that resolves with the decrypted snap state or null if no state exists.
-   * @throws If the snap state decryption fails.
+   * @param encrypted - A flag to indicate whether to use encrypted storage or not.
+   * @returns The requested snap state or null if no state exists.
    */
-  async getSnapState(snapId: ValidatedSnapId): Promise<Json> {
-    const state = this.state.snapStates[snapId];
+  getSnapState(snapId: ValidatedSnapId, encrypted: boolean): Json {
```

Comment:
Why was this async before?

Reply:
Probably from when the encryption was all handled in the SnapController